Description: Import LDFLAGS for hardening
 This patch fixes the hardening-no-relro Lintian warning by appending the
 LDFLAGS from dpkg-buildflags to the LDOPTIONS variable used by imake.
 See https://wiki.debian.org/Hardening for more information.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>

--- wmclock-1.0.14.orig/Imakefile
+++ wmclock-1.0.14/Imakefile
@@ -7,6 +7,8 @@ CFLAGS         += -Wall -pedantic -Wshad
 CFLAGS         += -Wcast-qual -Wchar-subscripts -Wpointer-arith 
 CFLAGS         += -Wredundant-decls -Wstrict-prototypes
 
+LDOPTIONS      += $(LDFLAGS)
+
 XPMLIB		= -L$(PREFIX)/lib/X11 -lXpm   
 DEPLIBS		= $(DEPXLIB) 
 
